src/queries/kv.web.ts

```diff
@@ -0,0 +1,24 @@
+import {useSuspenseQuery} from '@tanstack/react-query';
+import type {Chat} from '../lib/types';
+
+// TODO - This is a stub. Implement the getByKey query.
+export const getByKey = async <T>(key: string) => {
+  if (key === 'betas') {
+    return Promise.resolve(['beta-1', 'beta-2', 'beta-3'] as T);
+  }
+  if (key.startsWith('chat_')) {
+    return Promise.resolve({id: 1, title: 'Test - Chat Title'} as T);
+  }
+};
+
+export const useKey = <T>(key: string) => {
```

Comment:
I think these should stay in a platform agnostic file - only 'getByKey' and equivalents should be extracted out.

Reply:
yeah I agree, let's keep the platform code as small as possible